package.json

```diff
@@ -20,7 +20,7 @@
     "bootstrap": "yarn example && yarn && yarn pods",
     "bootstrap-no-pods": "yarn example && yarn",
     "docs": "yarn typedoc ./src/index.tsx --out ./docs/api-reference --tsconfig ./tsconfig.json --readme none --sort source-order",
-    "run-example-ios": "cd example;react-native run-ios --configuration Release --simulator \"iPhone 15\"",
+    "run-example-ios": "cd example;react-native run-ios --configuration Release --simulator \"iPhone 15 (17.2)\"",
```

Comment:
The default simulator (iPhone 15, iOS 17) was unreliable. iOS 17.2 seems much more stable.